deface/deface.py

```diff
@@ -366,40 +607,50 @@ def main():
     mosaicsize = args.mosaicsize
     keep_metadata = args.keep_metadata
     replaceimg = None
+    thresholds_by_sec = literal_eval(args.thresholds_by_sec)
+    overlap_threshold = args.overlap_threshold
+
     if in_shape is not None:
-        w, h = in_shape.split('x')
+        w, h = in_shape.split("x")
         in_shape = int(w), int(h)
     if replacewith == "img":
         replaceimg = imageio.imread(args.replaceimg)
-        print(f'After opening {args.replaceimg} shape: {replaceimg.shape}')
-
+        print(f"After opening {args.replaceimg} shape: {replaceimg.shape}")
 
     # TODO: scalar downscaling setting (-> in_shape), preserving aspect ratio
-    centerface = CenterFace(in_shape=in_shape, backend=backend, override_execution_provider=execution_provider)
+    centerface = CenterFace(
+        in_shape=in_shape,
+        backend=backend,
+        override_execution_provider=execution_provider,
+    )
 
     multi_file = len(ipaths) > 1
     if multi_file:
-        ipaths = tqdm.tqdm(ipaths, position=0, dynamic_ncols=True, desc='Batch progress')
+        ipaths = tqdm.tqdm(
+            ipaths, position=0, dynamic_ncols=True, desc="Batch progress"
+        )
 
     for ipath in ipaths:
         opath = base_opath
-        if ipath == 'cam':
-            ipath = '<video0>'
+        if ipath == "cam":
+            ipath = "<video0>"
             enable_preview = True
         filetype = get_file_type(ipath)
-        is_cam = filetype == 'cam'
+        is_cam = filetype == "cam"
         if opath is None and not is_cam:
             root, ext = os.path.splitext(ipath)
-            opath = f'{root}_anonymized{ext}'
-        print(f'Input:  {ipath}\nOutput: {opath}')
+            opath = f"{root}_anonymized{ext}"
+        print(f"Input:  {ipath}\nOutput: {opath}")
         if opath is None and not enable_preview:
-            print('No output file is specified and the preview GUI is disabled. No output will be produced.')
-        if filetype == 'video' or is_cam:
+            print(
+                "No output file is specified and the preview GUI is disabled. No output will be produced."
+            )
+        if filetype == "video" or is_cam:
             video_detect(
                 ipath=ipath,
                 opath=opath,
                 centerface=centerface,
-                threshold=threshold,
+                default_threshold=threshold,
```

Comment:
This is an interface-breaking change, which means, for libraries, a major version change (not sure if upstream respects that, though).
I think it would probably be better to not change that parameter's name, since it does clarifying things that much anyway.